packages/colorpicker/src/elements/ColorPicker/index.tsx

```diff
@@ -0,0 +1,193 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useEffect, useReducer, forwardRef, ChangeEvent, HTMLAttributes } from 'react';
+import PropTypes from 'prop-types';
+import { Saturation } from './Saturation';
+import {
+  StyledHue,
+  StyledSliderGroup,
+  StyledInput,
+  StyledLabel,
+  StyledAlpha,
+  StyledSliders,
+  StyledHexField,
+  StyledRGBAField,
+  StyledInputGroup,
+  StyledPreview,
+  StyledColorPicker,
+  StyledCheckered,
+  StyledAlphaGradient,
+  StyledAlphaField,
+  StyledHueField
+} from '../../styled';
+import { Label } from '@zendeskgarden/react-forms';
+import { getInitialState, reducer, IRGBColor, IHSVColor, IColorPickerState } from './reducer';
+
+export interface IColorPickerLabels {
+  hueSlider?: string;
+  alphaSlider?: string;
+  hex?: string;
+  red?: string;
+  green?: string;
+  blue?: string;
+  alpha?: string;
+}
+
+export interface IColorPickerProps
+  extends Omit<HTMLAttributes<HTMLDivElement>, 'color' | 'onChange'> {
+  /** A hex string, RGB string, RGB object, or color picker state that represents the current color */
+  color: string | IRGBColor | IColorPickerState;
+  /** A callback that is fired when an the color picker state is changed */
+  onChange?: (state: IColorPickerState) => void;
+  /** Replaces the default labels within the color picker */
+  labels?: IColorPickerLabels;
+}
+
+/**
+ * @extends HTMLAttributes<HTMLDivElement>
+ */
+export const ColorPicker = forwardRef<HTMLDivElement, IColorPickerProps>(
+  ({ color, labels = {}, onChange, ...props }, ref) => {
+    const [state, dispatch] = useReducer(reducer, getInitialState(color));
+
+    useEffect(() => {
+      onChange && onChange(state);
+    }, [state, onChange]);
+
+    return (
+      <StyledColorPicker ref={ref} {...props}>
+        <Saturation
```

Comment:
Can we use @steue's specified `ColorWell` terminology (throughout) rather than `Saturation`?

Reply:
Yeah, updated to use `ColorWell` terminology.